lib/generators/rolemodel/lograge/templates/config/initializers/lograge.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+unless %w[development test].include?(Rails.env)
```

Comment:
Could we use `exclude?`?

```suggestion
if %w[development test].exclude?(Rails.env)
```